### PR TITLE
OSDOCS-3826: Updated the compute section

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/OSDOCS-3826

Link to docs preview:

- ROSA:
    - [Compute Section](http://file.rdu.redhat.com/eponvell/OSDOCS-3826_Workload-Clarity/rosa/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-compute_rosa-service-definition)
    - [Red Hat Operator Support](http://file.rdu.redhat.com/eponvell/OSDOCS-3826_Workload-Clarity/rosa/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-red-hat-operator_rosa-service-definition)
- OSD:
    - [Compute Section](http://file.rdu.redhat.com/eponvell/OSDOCS-3826_Workload-Clarity/dedicated/osd_architecture/osd_policy/osd-service-definition.html#compute_osd-service-definition)
    - [Red Hat Operator Support](http://file.rdu.redhat.com/eponvell/OSDOCS-3826_Workload-Clarity/dedicated/osd_architecture/osd_policy/osd-service-definition.html#red-hat-operator-support)
- The following links fix #47807:
    - [Non-CCS Compute Types](http://file.rdu.redhat.com/eponvell/OSDOCS-3826_Workload-Clarity/dedicated/osd_architecture/osd_policy/osd-service-definition.html#aws-compute-types-non-ccs_osd-service-definition)
    - [GCP Compute Types  ](http://file.rdu.redhat.com/eponvell/OSDOCS-3826_Workload-Clarity/dedicated/osd_architecture/osd_policy/osd-service-definition.html#gcp-compute-types_osd-service-definition)

Additional information:
Updated the compute section in OSD and ROSA to note that no workloads should be deployed to Control plane and infrastructure nodes.

### CODE REVIEW

File: modules/rosa-sdpolicy-account-management.adoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ All {product-title} clusters support a maximum of 180 worker nodes.
...
+Control plane and infrastructure nodes are deployed and managed by Red Hat. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss. There are at least 3 control plane nodes that handle etcd- and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes, including Red Hat workloads, such as Red Hat Quay, Red Hat Advanced Cluster Management, Red Hat Advanced Cluster Security, Red Hat OpenShift Service Mesh, Red Hat OpenShift Serverless, Red Hat OpenShift Logging, Red Hat OpenShift Pipelines, etc.
```
Comment (stevsmit) [inline modules/rosa-sdpolicy-account-management.adoc:48]: Should we use the attributes here? 

For example, :pipelines-title: for Red Hat OpenShift Pipelines?

File: modules/rosa-sdpolicy-am-compute.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Multiple availability zone clusters require a minimum of 3 control planes. 3 infrastructure nodes, and 3 worker nodes. Additional nodes must be purchased in multiples of three to maintain proper node distribution.
+
+All {product-title} clusters support a maximum of 180 worker nodes.
+
+[NOTE]
+====
+The `Default` machine pool node type and size cannot be changed after the cluster is created.
+====
+
+Control plane and infrastructure nodes are deployed and managed by Red Hat. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss. There are at least 3 control plane nodes that handle etcd- and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes. See the additional resources for a list of the Red Hat Operator workloads.
```
Comment (wgordon17) [inline modules/rosa-sdpolicy-am-compute.adoc:20]:
```suggestion
Control plane and infrastructure nodes are deployed and managed by Red Hat. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss. There are at least 3 control plane nodes that handle etcd- and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes. See the Red Hat Operator support section below for more information on Red Hat workloads that must be deployed on worker nodes.
```

File: modules/sdpolicy-am-aws-compute-types-non-ccs.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+// Module included in the following assemblies:
+//
+// * assemblies/osd-service-definition.adoc
+:_content-type: CONCEPT
+[id="aws-compute-types-non-ccs_{context}"]
+= AWS compute types for standard clusters
+
+{product-title} offers the following worker node types and sizes on AWS:
+
+.General purpose
```
Comment (wgordon17) [inline modules/sdpolicy-am-aws-compute-types-non-ccs.adoc:11]: Perhaps you can incorporate this PR here: https://github.com/openshift/openshift-docs/pull/47807

File: modules/sdpolicy-am-compute.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Multiple availability zone clusters require a minimum of 3 worker nodes for Customer Cloud Subscription (CCS) clusters, 1 deployed to each of 3 availability zones. A minimum of 9 worker nodes are required for standard clusters. These 9 worker nodes are included in the base subscription, and additional nodes must be purchased in multiples of 3 to maintain proper node distribution.
+
+Worker nodes must all be the same type and size within a single {product-title} cluster.
+
+[NOTE]
+====
+The default machine pool node type and size cannot be changed after the cluster has been created.
+====
+
+Control plane and infrastructure nodes are also provided by Red Hat. There are at least 3 control plane nodes that handle etcd and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control plane and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes. See the additional resources for a list of the Red Hat Operator workloads.
```
Comment (wgordon17) [inline modules/sdpolicy-am-compute.adoc:20]:
```suggestion
Control plane and infrastructure nodes are also provided by Red Hat. There are at least 3 control plane nodes that handle etcd and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control plane and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes. See the Red Hat Operator support section below for more information on Red Hat workloads that must be deployed on worker nodes.
```

File: modules/sdpolicy-platform.adoc [hunk 5]
```diff
@@ -90,6 +90,18 @@ Windows containers are not available on {product-title} at this time.
 == Operating system
 {product-title} runs on OpenShift 4 and uses Red Hat Enterprise Linux CoreOS as the operating system for all control plane and worker nodes.
 
+== Red Hat Operator support
+[id="sdpolicy-red-hat-operator_{context}"]
+Red Hat workloads typically refer to any Red Hat Operator. These Operators may require additional Red Hat subscriptions, and may incur additional cloud infrastructure costs. Some of the Red Hat Operators are:
```
Comment (wgordon17) [inline modules/sdpolicy-platform.adoc:95]:
```suggestion
Red Hat workloads typically refer to Red Hat-provided operators made available through Operator Hub. Red Hat workloads are not managed by the Red Hat SRE team, and must be deployed on worker nodes. These operators may require additional Red Hat subscriptions, and may incur additional cloud infrastructure costs. Examples of these Red Hat-provided operators are:
```

File: modules/rosa-sdpolicy-platform.adoc [hunk 6]
```diff
@@ -98,6 +98,18 @@ See the link:https://docs.openshift.com/rosa/rosa_policy/rosa-life-cycle.html[{p
 == Operating system
 {product-title} runs on OpenShift 4 and uses Red Hat CoreOS as the operating system for all control plane and worker nodes.
 
+[id="rosa-sdpolicy-red-hat-operator_{context}"]
+== Red Hat Operator support
+Red Hat workloads typically refer to any Red Hat Operator. These Operators may require additional Red Hat subscriptions, and may incur additional cloud infrastructure costs. Some of the Red Hat Operators are:
```
Comment (wgordon17) [inline modules/rosa-sdpolicy-platform.adoc:103]:
```suggestion
Red Hat workloads typically refer to Red Hat-provided operators made available through Operator Hub. Red Hat workloads are not managed by the Red Hat SRE team, and must be deployed on worker nodes. These operators may require additional Red Hat subscriptions, and may incur additional cloud infrastructure costs. Examples of these Red Hat-provided operators are:
```

**Review by wgordon17 — CHANGES_REQUESTED** [review]

**Review by wgordon17 — APPROVED** [review]
LGTM 👍

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**